drivers/SmartThings/zigbee-lock/src/init.lua

```diff
@@ -300,6 +304,92 @@ local function init(driver, device)
   lock_utils.populate_state_from_data(device)
 end
 
+-- The following two functions are from the lock defaults
+-- They are in the base driver temporarily because of BUG2-659
```

Comment:
nit: only internal devs know what BUG2-659 is, the comment could be more useful for the community